cylc/flow/scheduler_cli.py

```diff
@@ -523,6 +523,7 @@ def _version_check(
                     process=str.lower,
                 )
             # we are in non-interactive mode, abort abort abort
+            print(cparse('Use "--upgrade" to upgrade the workflow.'))
```

Comment:
* This message should not appear in interactive mode (covered by the prompts above).
* No need for `cparse` unless ANSI colour is used in the message.

```suggestion
            else:
                print('Use "--upgrade" to upgrade the workflow.')
```